coralproject/talk
# Added count clearing for status changes

## What does this PR do?

Resolves issue where comment counts aren't cleared when updated.

## How do I test this PR?

1. Create a comment
2. Reload the page to reload the comment count
3. Flag the comment
4. Go into the admin, reject the comment
5. Go back to the stream, notice that the stream count is correct
